### PR TITLE
Backport Inspect Address

### PR DESCRIPTION
**Pull request by ritesh-skyach**
Backport Inspect Address

**Issue :**

- https://jira.openware.work/browse/PEATIO-74

### CODE REVIEW

File: lib/blockchain_client/ripple.rb [hunk 1]
```diff
@@ -4,5 +4,16 @@
 module BlockchainClient
   class Ripple < Base
```
Comment (ysv) [inline lib/blockchain_client/ripple.rb]: @ritesh-skyach Remove ripple stuff it's pending temporary

**Review by ysv — CHANGES_REQUESTED** [review]

**Comment by ysv** [comment]
@ritesh-skyach Backport specs also
https://github.com/rubykube/peatio/pull/1542/files#diff-724bad691eb6fc1d25faf37e8a9c0a4b

File: lib/blockchain_client/ripple.rb (hunk 1 above)
Comment (dinesh-skyach) [reply lib/blockchain_client/ripple.rb]: @ysv can't remove this, as this fails spec in some part.

**Comment by mod** [comment]
conflict to solve

**Comment by ritesh-skyach** [comment]
@mod Conflicts Resolved

**Pull request merged**